src/clj/puppetlabs/puppetserver/jruby_request.clj

```diff
@@ -67,10 +67,10 @@
      (handler (assoc request :jruby-instance jruby-instance)))))
 
 (defn get-environment-from-request
-  "Gets the environment from a request."
+  "Gets the environment from the URL or query string of a request."
   [req]
-  (-> req
-      (get-in [:params "environment"])))
+  (or (get-in req [:params :environment])
+      (get-in req [:params "environment"])))
```

Comment:
@camlow325 what would that look like?  Digging into the less generic params maps in the request?  Like... `route-params` and `query-params`?  If that's what you're suggesting that sounds like it might be worthwhile.  Does `stringify-keys` monkey will all of those maps or just the generic `params` map?